examples/advanced/autowrap_ufuncify.py

```diff
@@ -24,11 +24,14 @@
 np = import_module('numpy')
 if not np:
     sys.exit("Cannot import numpy. Exiting.")
+plt = import_module('matplotlib.pyplot')
```

Comment:
I don't see any extra commit.
Anyway, you need to properly merge master. There shouldn't be any extra changes.